optionals/compat_rhs_afrf3/CfgWeapons.hpp

```diff
@@ -57,6 +57,10 @@ class CfgWeapons {
     class rhs_weap_svds_npz: rhs_weap_svds {
         ACE_RailHeightAboveBore = 4.3348;
     };
+    class rhs_weap_rpk74_base;
+    class rhs_weap_rpk74m : rhs_weap_rpk74_base {
```

Comment:
After looking at their configs I think `class rhs_weap_rpk74_base: rhs_weap_rpk_base` would be a better choice
and probably keep the `ACE_barrelTwist = 195.072;` config from before as well

Reply:
Changed in newest commit, tested it and it works for me too